raven-android/src/main/java/com/getsentry/raven/android/Util.java

```diff
@@ -60,4 +64,19 @@ public static boolean shouldAttemptToSend(Context ctx) {
         return isConnected(ctx);
     }
 
+    /**
+     * Reads a line from the specified file.
+     * @param filename the file to read from
+     * @return the first line, if any.
+     * @throws IOException if the file couldn't be read
+     */
+    public static String readLine(String filename) throws IOException {
```

Comment:
It seems weird to me to have a public utility method for this since it's only going to be useful in the one place. May as well just do do this code in `getKernelVersion`?